clang/lib/DPCT/CUBAPIMigration.cpp

```diff
@@ -252,9 +254,17 @@ void CubMemberCallRule::runRule(
         Name == "BlockedToStriped" || Name == "StripedToBlocked" ||
         Name == "StripedToBlocked" || Name == "ScatterToBlocked" ||
         Name == "ScatterToStriped";
-    if (isBlockRadixSort || isBlockExchange) {
-      std::string HelpFuncName =
-          isBlockRadixSort ? "group_radix_sort" : "exchange";
+    if (isBlockRadixSort || isBlockExchange || Name == "Load" ||
+        Name == "Store") {
+      std::string HelpFuncName;
+      if (isBlockRadixSort)
```

Comment:
may use a unordered map to do this?

Reply:
I don't think so. inline string has more optimization opportunities.